code/modules/mob/living/living_defines.dm

```diff
@@ -12,8 +12,10 @@
 	var/lastattackerckey = null
 
 	//Health and life related vars
-	var/maxHealth = 100 ///Maximum health that should be possible.
-	var/health = 100 	///A mob's health
+	/// Maximum health that should be possible.
+	var/maxHealth = MAX_LIVING_HEALTH
+	/// The mob's current health.
+	var/health = MAX_LIVING_HEALTH
```

Comment:
this should be in a separate PR.

Reply:
It will conflict with the removal of `human_maxHealth`, but sure.